src/accessors/IEnvironmentRead.ts

```diff
@@ -0,0 +1,10 @@
+import { ISettingRead } from './ISettingRead';
+/**
+ * Allows read-access to configuration and other data which is not created by the normal user
+ */
+
+export interface IEnvironmentRead{
+    settings(): ISettingRead;
+
+    environmentVariables(): object; //todo
```

Comment:
I do also think of a subset. Again, this would be an accessor (`IEnvironmentVariablesRead`) with a `getter` which could transparently handle the variables' exposure